.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+<!--
+Please make sure you've read and understood our contributing guidelines;
+https://github.com/checkpoint-restore/criu/blob/criu-dev/CONTRIBUTING.md
+
+In short you need to:
+
+- Describe What you do and How you do it;
+- Separate each logical change into a separate commit;
+- Sign your work;
```

Comment:
Good idea. The only thing I would change it the `Sign your work` part to make it clearer we mean Signed-off-by and maybe mention the DCO more explicitly.

Reply:
It was hard to make it both short and self-explaining. Hope this update would be ok:

```
-- Sign your work;
+- Add a "Signed-off-by:" line identifying that you certify your work with DCO;
```